oracledb-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,1440 @@
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+  targets: [
+    {
+      datasource: promDatasource,
+      editorMode: 'code',
+      expr: 'oracledb_tablespace_bytes{instance=~"$instance",job=~"$job",tablespace=~"$tablespace"}',
+      legendFormat: '{{instance}} - {{tablespace}} - used',
+      range: true,
+      refId: 'Used Bytes',
+    },
+    {
+      datasource: promDatasource,
+      editorMode: 'code',
+      expr: 'oracledb_tablespace_free{instance=~"$instance",job=~"$job",tablespace=~"$tablespace"}',
+      hide: false,
+      legendFormat: '{{instance}} - {{tablespace}} - free',
+      range: true,
+      refId: 'Free',
+    },
+    {
+      datasource: promDatasource,
+      editorMode: 'code',
+      expr: 'oracledb_tablespace_max_bytes{instance=~"$instance",job=~"$job",tablespace=~"$tablespace"}',
+      hide: false,
+      legendFormat: '{{instance}} - {{tablespace}} - max',
+      range: true,
+      refId: 'Max',
+    },
+  ],
+  title: 'Tablespace Size',
+  type: 'timeseries',
+};
+
+{
+  grafanaDashboards+:: {
+    'oracledb-overview.json':
+      dashboard.new(
+        'OracleDB Overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        editable=true,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        graphTooltip='shared_crosshair',
+        uid=dashboardUid,
+      )
+      .addLink(grafana.link.dashboards(
+        title='Other OracleDB dashboards',
+        includeVars=true,
+        keepTime=true,
+        tags=($._config.dashboardTags),
+      ))
+      .addTemplates(
+        [
+          {
+            hide: 0,
+            label: 'Data source',
+            name: promDatasourceName,
+            query: 'prometheus',
+            refresh: 1,
+            regex: '',
+            type: 'datasource',
+          },
+          {
+            hide: 0,
+            label: 'Loki datasource',
+            name: 'loki_datasource',
+            query: 'loki',
+            refresh: 1,
+            regex: '',
+            type: 'datasource',
+          },
+          template.new(
+            name='instance',
+            label='Instance',
+            datasource='$prometheus_datasource',
+            query='label_values(oracledb_up, instance)',
+            current='',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='.+',
+            sort=1
+          ),
+          template.new(
+            'job',
+            promDatasource,
+            query='label_values(oracledb_up, job)',
+            label='Job',
+            refresh='time',
+            includeAll=true,
+            multi=true,
+            allValues='.+',
+            sort=1
+          ),
+          template.new(
+            'tablespace',
+            promDatasource,
+            query='label_values(oracledb_tablespace_bytes, tablespace)',
```

Comment:
Not 100% on this: Should tablespace be constrained with to just available tablespaces for the given `instance` and `job` selectors, or are those labels not available in `oracledb_tablespace_bytes`?

Reply:
Ahh good catch, made that change 👍